akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -170,6 +170,17 @@ object ClusterEvent {
   }
 
   /**
+   * This event is published when the cluster node is shutting down,
+   * before the final [[MemberRemoved]] events are published.
+   */
+  case object ClusterShuttingDown extends ClusterDomainEvent
```

Comment:
yes, that is the intended usage. we don't have custom serialization or serialversionuid for any of these ClusterDomainEvent